src/lib/utils/sizes.js

```diff
@@ -29,14 +30,16 @@ const sizes = new class {
     const width = getScreenWidth()
     const { maxWidthForTabletAndDesktop } = theme.sizes
 
-    return Math.min(width, maxWidthForTabletAndDesktop)
+    // To ensure it doesn't return values smaller than device width on mobile
+    return isMobileNative ? width : Math.min(width, maxWidthForTabletAndDesktop)
```

Comment:
again  i think this should be applied to isMobile not isMobileNative

Reply:
@julianpolcode please also re check is this fixes the web app running on web browser on such devices
you could use ngrok.io to serve localhost:3000 via https over some public domain (to easily test local web version on mobile)

Reply:
I've checked mobile, desktop and native